cmd/yurthub/app/config/config.go

```diff
@@ -251,10 +252,14 @@ func buildProxiedClient(proxyAddr string) (kubernetes.Interface, error) {
 }
 
 // createSharedInformers create sharedInformers from the given proxyAddr.
-func createYurtSharedInformers(proxiedClient kubernetes.Interface, enableNodePool bool) (yurtinformers.SharedInformerFactory, error) {
+func createYurtSharedInformers(proxyAddr string, enableNodePool bool) (yurtinformers.SharedInformerFactory, error) {
 	var kubeConfig *rest.Config
```

Comment:
kubeConfig would cause nil pointer  in line:
```go
if enableNodePool {
		yurtClient, err = yurtclientset.NewForConfig(kubeConfig)
		if err != nil {
			return nil, err
		}
	}

```

Reply:
Could we just
```go
	if enableNodePool {
		yurtClient = proxiedClient
	}
```

Reply:
Ohh, it's merged, I did not fresh my page. I think it's fine to keep it as it is since it also works.